deployment/serverless.yml

```diff
@@ -78,7 +78,7 @@ custom:
 functions:
   change-event-dlq-handler:
     image: ${env:DOCKER_REGISTRY}/${env:CHANGE_EVENT_DLQ_HANDLER}:${env:VERSION}
-    architecture: x86_64
+    architecture: arm64
```

Comment:
Most minor thing, as you have architecture in the top configuration it should automatically be passed down to the lambda configuration so you don't need to set architecture on each lambda